flask_pydantic_spec/spec.py

```diff
@@ -191,10 +236,10 @@ def _generate_spec(self) -> Mapping[str, Any]:
         tag_lookup = {tag["name"]: tag for tag in self.config.TAGS}
         routes: Dict[str, Any] = {}
         tags: Dict[str, Any] = {}
-        for route in self.backend.find_routes():
+        for route in self.backend.find_routes(self.app):
```

Comment:
What happens if `self.app` is `None`? Do we need to double check and bail with a useful error at this point or is that already handled further up the stack?